### DISCUSSION TITLE
listenerMiddleware.startListening.withTypes type error: Property withTypes doesn't exist

### DISCUSSION THREAD

**Discussion by mattsbennett** [reduxjs/redux-toolkit#4085] [Q&A] [answered]
I'm trying to follow the docs example here: https://redux-toolkit.js.org/api/createListenerMiddleware#typescript-usage

However, the example code does not compile due to a type error - apparently the `withTypes` property doesn't exist. Is this a bug or am I doing something wrong?

Here is a minimal example: https://codesandbox.io/p/devbox/435m6c?file=%2Fsrc%2Fstore.ts%3A9%2C27-9%2C70

**Comment by markerikson** [comment]
@mattsbennett We've merged the PR to `master`, so the docs got updated, but we haven't put out a new release yet containing those changes :)

**Comment by EskiMojo14** [comment, marked as answer]
yet to be released in 2.1.0 (likely this week), sorry - docs are automatically built from `master`

**Reply by EskiMojo14** [reply]
in the meantime, the previous usage looks like 
```ts
import { createListenerMiddleware, addListener, TypedStartListening, TypedAddListener } from '@reduxjs/toolkit'
import type { RootState, AppDispatch } from './store'

export const listenerMiddleware = createListenerMiddleware()

export const startAppListening = listenerMiddleware.startListening as TypedStartListening<RootState, AppDispatch>

export const addAppListener = addListener as TypedAddListener<RootState, AppDispatch>
```

**Reply by mattsbennett** [reply]
Wonderful, thank you 🙏

**Comment by mattsbennett** [comment]
Thanks @markerikson @EskiMojo14
